docs/CONFIGURATION.md

```diff
@@ -85,9 +87,12 @@ alertmanager:
   TLS connections to this Alertmanager instance if it requires a TLS client
   authentication.
   Note that this option requires `tls:cert` to be also set.
+- `customheaders` - a map with a list of key: values which are header: value.
```

Comment:
you have `customheaders` here and `customHeaders` above, calling it just `headers` would probably help with typos like this